cypress/integration/bulk-edit/bulk-edit-in-app-holdings-uploading.spec.js

```diff
@@ -0,0 +1,60 @@
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import FileManager from '../../support/utils/fileManager';
+import getRandomPostfix from '../../support/utils/stringTools';
+import testTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import BulkEditSearchPane from '../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import BulkEditActions from '../../support/fragments/bulk-edit/bulk-edit-actions';
+import BulkEditFiles from '../../support/fragments/bulk-edit/bulk-edit-files';
+import Users from '../../support/fragments/users/users';
+
+let user;
+const itemBarcode = getRandomPostfix();
+const validHoldingUUIDsFileName = `validHoldingUUIDs_${getRandomPostfix()}.csv`;
+const resultFileName = `matchedRecords_${getRandomPostfix()}.csv`;
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode1: itemBarcode,
+  itemBarcode2: `secondBarcode_${itemBarcode}`
+};
+
+describe('bulk-edit: in-app file uploading', () => {
+  before('create user', () => {
+    cy.createTempUser([
+      permissions.bulkEditView.gui,
+      permissions.bulkEditEdit.gui,
+    ])
+      .then(userProperties => {
+        user = userProperties;
+        cy.login(user.username, user.password);
+        cy.visit(TopMenu.bulkEditPath);
```

Comment:
Please use the the path parameter in the login function instead of cy.visit

Reply:
because the login function have this code fragment in the implementation  cy.visit(visitPath.path); . So we have cy.visit(/)
and then cy.visit(TopMenu.bulkEditPath). It means one additional usage of this function and it also takes redundant time to open the page.

Reply:
done